cx_Freeze/freezer.py

```diff
@@ -162,76 +138,26 @@ def _CopyFile(
+            source=source, target=target,
+            normalizedSource=normalizedSource,
+            normalizedTarget=normalizedTarget,
+            copyDependentFiles=copyDependentFiles,
+            includeMode=includeMode
+        )
 
-        if (
-            copyDependentFiles
-            and source not in self.finder.exclude_dependent_files
-        ):
-            # Always copy dependent files on root directory
-            # to allow to set relative reference
-            if sys.platform == "darwin":
-                targetdir = self.targetdir
-                for dependent_file in self._GetDependentFiles(
-                    source, darwinFile=newDarwinFile
-                ):
-                    target = os.path.join(
-                        targetdir, os.path.basename(dependent_file)
-                    )
-                    self._CopyFile(
-                        dependent_file,
-                        target,
-                        copyDependentFiles=True,
-                        machOReference=newDarwinFile.getMachOReferenceForPath(
-                            path=dependent_file
-                        ),
-                    )
-            elif sys.platform == "win32":
-                for dependent_file in self._GetDependentFiles(source):
-                    target = os.path.join(
-                        targetdir, os.path.basename(dependent_file)
-                    )
-                    self._CopyFile(dependent_file, target, copyDependentFiles)
-            else:
-                source_dir = os.path.dirname(source)
-                library_dir = os.path.join(self.targetdir, "lib")
-                fix_rpath = set()
-                for dependent_file in self._GetDependentFiles(source):
-                    dep_base = os.path.basename(dependent_file)
-                    dep_abs = os.path.abspath(dependent_file)
-                    dep_rel = os.path.relpath(dep_abs, source_dir)
-                    if targetdir == library_dir:
-                        while dep_rel.startswith(os.pardir + os.sep):
-                            dep_rel = dep_rel[len(os.pardir + os.sep) :]
-                    dep_libs = dep_rel[: -(len(dep_base) + 1)]
-                    if dep_libs:
-                        fix_rpath.add(dep_libs)
-                    dependent_target = os.path.join(targetdir, dep_rel)
-                    self._CopyFile(
-                        dependent_file,
-                        dependent_target,
-                        copyDependentFiles,
-                    )
-                if fix_rpath:
-                    has_rpath = self.patchelf.get_rpath(target)
-                    rpath = ":".join([f"$ORIGIN/{r}" for r in fix_rpath])
-                    if has_rpath != rpath:
-                        self.patchelf.set_rpath(target, rpath)
+    def _CopyFileNameHook(self, source: str, normalizedTarget: str) -> str:
+        return normalizedTarget
+
+    @abstractmethod
+    def _PostCopyHook(self,
```

Comment:
As you know I have been trying to transform the code little by little into pep8. So, new functions can be in pep8. In this case _post_copy_hook. By the way, why did you adopt the 'hook'? I am not against (nor for), it is just to know.

Reply:
I tried to use "hook" where some of the platforms might have wanted to run some extra code.  It ended up not being entirely consistent after all the changes I made. Once everything else is settled, it will be easy enough to change all the names to a consistent scheme. 

I'm happy to convert all the functions in Freezer to pep8 names, while I'm at it.  I do not see any downside to doing that, at least for all the private functions.  Let me know.

Reply:
Okay, makes sense.